bigquery/src/main/java/org/apache/zeppelin/bigquery/bigQueryInterpreter.java

```diff
@@ -0,0 +1,340 @@
+/**
+ * Copyright 2016 Google Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
```

Comment:
I would strongly prefer keeping original text of the license here: `See the NOTICE file distributed with this work for additional information regarding copyright ownership.` 

So copyright notice need to be moved to root NOTICE file and refer to `./bigquery/` module.